### ISSUE TITLE
Josh Fermin

### ISSUE THREAD

**Issue by joshfermin** [antsankov/data-engineering-HW1#9]
I liked how you used mongo and python in this assignment. The implementation was easy to read through in terms of looking at "rest.py", and I was able to get everything running easily. A couple things you could do is add more documentation to the README.md about what you can do with the different routes. I had to look into rest.py to see what I could do. 

Also, one other thing I noticed is some sort of connection key to mongolabs in your rest.py file on line 4:
```python 
connection = Connection("15bcI8yedImO3qWvB6bZncbz_v3VONO3")
```
I think it might be best to keep this key out of source control, so that others will not have access to it. Maybe adding it to a different file and then adding that file to your .gitignore would be one option. But then again I am not too clear on what this key does, but it might be best to keep it out of the source control.

**Comment by antsankov** [comment]
It's kind of a chicken and egg situation, because if we hide it, then the person running it won't be able to connect to the DB and the service won't work. 

I've asked the [developers of the API this very question](https://github.com/puentesarrin/pymongolab/issues/6) about the proper way to hide the key in this type of scenario, so I'll make any changes if they recommend anything.

**Comment by joco1508** [comment]
I have added more documentation to the readme and general information about the routes. Hope it helps!

**Comment by kenbod** [comment]
Yes, in general, you would want to keep the key out of your source code; for this assignment, it's fine. One approach you can use for production systems is to have the code written such that the key is stored in a properties file that either lives in a well known location or is passed in via the command line. Then in your README, you provide instructions for how a user can get their own key and then have them use your service with their own database.

I have to admit that I don't know much about Mongo Labs. Would someone want to do a presentation on the service it provides?
